.gitignore

```diff
@@ -20,3 +20,6 @@ erl_crash.dump
 *.ez
 
 /.elixir_ls
+
+# Emacs project file
+.projectile
```

Comment:
It would be awesome if @bryanjos is ok with this, however I think its better
to remove this change, because it is a personal thing, there are as many text editors as devs and it belongs into your own global `~/.gitignore` file.